compiler/test/compilable/extra-files/header2i.di

```diff
@@ -3,26 +3,29 @@
 class C
 {
 }
-void foo(const C c, const(char)[] s, const int* q, const(int*) p)
+void foo(const(C) c, const(char)[] s, const(int*) q, const(int*) p)
 {
 }
 void bar(in void* p)
 {
 }
 void f(void function() f2);
 class C2;
-void foo2(const C2 c);
+void foo2(const(C2) c);
 struct Foo3
 {
 	int k;
-	@nogc @live @trusted @disable ~this()
+	@disable ~this() @nogc @live @trusted
 	{
-		k = 1;
+		this.k = 1;
 	}
 	this(this)
 	{
-		k = 2;
+		this.k = 2;
 	}
+	alias __xdtor = __dtor;
+	alias __xpostblit = __postblit;
```

Comment:
do we really want these compiler internal names to be emitted? or is this simply how it comes out of the post semantic AST?

Also is aliasing these internal names still legal D?